### ISSUE TITLE
[FEATURE] Automatic creation of Calibre columns

### ISSUE THREAD

**Issue by WillScarlettOhara** [kyxap/koreader-calibre-plugin#50]
### Feature Summary

Automatic creation of Calibre columns

### Motivation

The creation of columns is not easy to understand and a little tedious.

### Use Cases

The first time I used this plugin, I was completely lost when it came to creating columns. Making this plugin easier to set up would certainly contribute to its popularity.

**Comment by kyxap** [comment]
yeah that wound be nice but it seems like requires to write into calibre db which make think much complicated

**Comment by WillScarlettOhara** [comment]
Kobo Utilities does this, but in a semi-automated way, as you can create directly from the plugin with the right parameters already filled in:
![image](https://github.com/user-attachments/assets/d86e667c-9128-4ee1-927a-507b429342bc)

**Comment by kyxap** [comment]
yeah that why they have this warning:
<img width="580" alt="image" src="https://github.com/user-attachments/assets/20477b4d-f1b0-4bb5-bb53-0fa92f98a8b3">

https://www.mobileread.com/forums/showthread.php?t=215339

**Comment by jbhul** [comment]
I've implemented a version of this on my fork on [this](https://github.com/jbhul/koreader-calibre-plugin/commit/b68c1330ef525174ebe72345391dc99fdc9432d1) commit. 

I used Goodreads Sync as a reference

It's a pretty big config refactor, but I tested it and everything looks fine on my end.

**Comment by kyxap** [comment]
Will be delivered as part of #71
